test/toolkits/style_transfer/test_style_transfer.cxx

```diff
@@ -16,8 +16,85 @@
 
 #include "utils.hpp"
 
+/**
+ * Since the only backend is MPS, these tests won't pass on any system other
+ * than 10.15.
+ */
+#ifdef HAS_MACOS_10_15
```

Comment:
Why do our unit tests for the backend-agnostic C++ ST code depend on any particular platform or backend?

Reply:
On linux there isn't a compute context for style transfer yet that's callable directly through C++. So the create_compute context will return a `nullptr` this will throw an exception. But since I can't figure out how to test that I took these tests out.

Reply:
Even the TensorFlow code is only callable through python because of PyBind11 dependencies. Not sure we can call that code directly through C++.

Reply:
Please look at the unit tests for OD, and the ones that @shantanuchhabra and @Jarvi-Izana are now adding for DC